spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -517,20 +566,140 @@ function chanCloseConfirm(
     connection = provableStore.get(connectionPath(channel.connectionHops[0]))
     abortTransactionUnless(connection !== null)
     abortTransactionUnless(connection.state === OPEN)
+
+    let counterpartyHops: []string = []
+    if (connectionHops.length > 1) {
+      counterpartyHops = getCounterPartyHops(proofInit, connection)
```

Comment:
Could you change `proofInit: CommitmentProof | MultihopProof,` in the params of `chanCloseConfirm` above?